qbft/spectest/tests/messages/round_change_prepared.go

```diff
@@ -1,8 +1,31 @@
 package messages
 
-import "github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
 
 // RoundChangePrepared tests a round change prepared return value
 func RoundChangePrepared() tests.SpecTest {
-	panic("implement")
+	ks := testingutils.Testing4SharesSet()
+
+	prepareMsgs := []*qbft.SignedMessage{
+		testingutils.TestingPrepareMessage(ks.Shares[1], types.OperatorID(1)),
+		testingutils.TestingPrepareMessage(ks.Shares[2], types.OperatorID(2)),
+		testingutils.TestingPrepareMessage(ks.Shares[3], types.OperatorID(3)),
+	}
+
+	prepareMarshalled := testingutils.MarshalJustifications(prepareMsgs)
+
+	msg := testingutils.TestingRoundChangeMessageWithParams(
+		ks.Shares[1], types.OperatorID(1), 2, qbft.FirstHeight, testingutils.TestingQBFTRootData, 1, prepareMarshalled)
+
+	return &tests.MsgSpecTest{
+		Name: "round change prepared",
+		Messages: []*qbft.SignedMessage{
+			msg,
+		},
+	}
 }
```

Comment:
I actually don't understand the point of this test. We're checking our `testingutils`?

I am not sure what test makes sense here, @alonmuroch?